app/views/local_transaction/unavailable_service.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= render layout: 'shared/base_page', locals: {
+  title: "This service is not available in #{@country_name}",
+  publication: @publication,
+  edition: @edition,
+} do %>
+  <div class="interaction">
+    <p class="govuk-body">
+      We’ve matched the postcode <span class="postcode"><%= @postcode %></span> to <span class="local-authority"><%= @local_authority.name %></span>.
+    </p>
+
+    <p class="govuk-body">
+      This service is not available in <span class="country-name"><%= @country_name %></span>. You can find other services on the <span class="local-authority"><%= @local_authority.name %></span> website.
+    </p>
+
+    <p id="get-started" class="get-started group">
+      <%= render "govuk_publishing_components/components/button", {
+        text: "Find other services",
+        rel: "external",
+        start: true,
```

Comment:
Think we can remove this - as its not a start button.

Reply:
See https://github.com/alphagov/frontend/pull/2550#discussion_r543416233

Reply:
Removed from `unavailable_service` page only